### PR TITLE
Pipeline fixes

### PR DESCRIPTION
**Pull request by zovomat**

### CODE REVIEW

**Comment by zovomat** [comment]
This is a ~~draft~~ pr for the CI changes, ~~I'm missing only the pr but it's almost done~~

File: Jenkinsfile [hunk 1]
```diff
@@ -228,42 +207,34 @@ pipeline {
...
-
-        stage('Release Beta') {
+        stage('Release Devel on NPM') {
             when {
                 beforeAgent true
                 allOf {
-                    expression { BRANCH_NAME ==~ /(beta)/ }
-                    environment name: 'COMMIT_PARENTS_COUNT', value: '1'
+                    expression { BRANCH_NAME ==~ /(devel)/ }
+                    environment name: 'COMMIT_PARENTS_COUNT', value: '2'
```
Comment (beawar) [inline Jenkinsfile]: Doesn't this check block possible squash commits in devel?

Comment (zovomat) [reply Jenkinsfile]: I forgot about that! I'll look for a better way to detect merges

**Review by giuliano176 — DISMISSED** [review]

File: Jenkinsfile (hunk 1 above)
Comment (zovomat) [reply Jenkinsfile]: after some trial and error, we concluded that it's probably better to publish the devel tag on any commit on the branch (it is push-protected anyway)

**Review by beawar — APPROVED** [review]

**Review by CataldoMazzilli — APPROVED** [review]

**Pull request merged**